src/main/java/data/Receipt.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/* Class to represent a receipt and its properties.*/
+public class Receipt {
+  private long id;
+  private long userId;
+  private long timestamp;
+  private BlobKey blobKey;
+  private String imageUrl;
+  private double price;
+  private String store;
+  private String label;
+  private Set<String> categories;
```

Comment:
Prefer `ImmutableSet` (see https://github.com/google/guava/wiki/ImmutableCollectionsExplained for advantages) if you don't plan to modify `categories` later.

In the constructor you can then do the following
```
this.categories = ImmutableSet.copyOf(categories);
```